gradle/dependencies.gradle

```diff
@@ -1,5 +1,5 @@
 def versions = [
-    apolloVersion    : '0.2.3-SNAPSHOT',
+    apolloVersion    : '0.3.0',
```

Comment:
been out of the loop but if 0.3.0 isn't out yet, this should stay as `0.2.3-SNAPSHOT` because it's used to pull dependencies from maven for the sample and apollo-integration.